test/test_s3_crawl.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+import os
+import tarfile
+
+import pytest
+from six.moves.urllib.parse import urlparse
+
+from ..automation import TaskManager
+from ..automation.utilities import db_utils, domain_utils
+from .openwpmtest import OpenWPMTest
+
+from .utilities import local_s3_bucket, download_s3_directory
+
+TEST_SITES = [
+    'http://example.com'
+]
+
+psl = domain_utils.get_psl()
+
+
+class TestS3Crawl(OpenWPMTest):
+    """ Runs a short test crawl.
+
+    This should be used to test any features that require real
+    crawl data. This should be avoided if possible, as controlled
+    tests will be easier to debug
+    """
+
+    def get_config(self, data_dir=""):
+        manager_params, browser_params = self.get_test_config(data_dir)
+        # Requires that `SERVICES=s3 localstack start` has been started
```

Comment:
Does this happen automatically? Is this something we expect the users to run before they stat the tests? If so, can we automate as part of the test suite -- perhaps as a test fixture?

Reply:
I see you are resolving this in https://github.com/mozilla/OpenWPM/pull/333, thanks :)